README.md

```diff
@@ -33,29 +33,36 @@ modified for specific [OpenLab](https://github.com/theopenlab) CI system deploym
 
      Using two nodes to  create a new CI system. One is for Master, the other one is for Slave
 
-   - New Slave Mode (`new-slave`)
+   - New Slave for HA Mode(`new-slave`)
 
-     Recovering a new Slave node based on the existing Master node. Before runing with this mode,
-     please check the `inventory/openlab-new-slave.yaml`, you may need to config the new slave
-     `nodepool` and `zuul` hosts IP, also you can choose whether to sync the old **zuul**
-     database to new slave or not.
+     There is not a specific inventory yaml file of this mode, you can use `--new-slave` argument
+     and new slave hsots IPs specified by `--new-ip` argument of the `deploy.py` tool. For an example,
```

Comment:
typo: 
hsots -> hosts
For an example -> For example?

Reply:
done, thx